src/main/webapp/i18n/de/student-dashboard.json

```
@@ -190,6 +190,8 @@
                 "rated": "Bewertet",
                 "practice": "Übung",
                 "assessmentType": "Bewertung:",
+                "startDate": "Startdatum:",
+                "startDateTooltip": "Student:innen können diese Aufgabe ab {{date}} bearbeiten.",
```

```suggestion
                "startDateTooltip": "Studierende können diese Aufgabe ab {{date}} bearbeiten.",
```
According to our guidelines.